### PR TITLE
Remedy port override bug fix

### PR DESCRIPTION
**Pull request by vitiwari**
@gokumar 
Remedy port override bug fix.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ $ mvn clean install
 $cd dist
 2. Change the incidentTemplate.json/changeTemplate.json configuration (based on description below)
 3. Run jar file
-$java -jar remedy-meter-script-0.0.1-SNAPSHOT-full.jar <incident> <change>
```
Comment (gokumar) [inline README.md:31]: Name of the jar file is different

File: README.md [hunk 2]
```diff
@@ -45,6 +45,7 @@ $java -jar remedy-meter-script-0.0.1-SNAPSHOT-full.jar <incident> <change>
 ```
 "config": {
   "remedyHostName":"xxxx"  					---> ARServer Host name
+  "remedyPort":0,                                               ---> ArServer TCP Port
```
Comment (gokumar) [inline README.md]: Are we going to add it by default in the config ? if not, remove it from here

**Review by gokumar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (vitiwari) [reply README.md:31]: It has been removed (see - sign).

File: README.md (hunk 2 above)
Comment (vitiwari) [reply README.md]: done

File: README.md (hunk 1 above)
Comment (gokumar) [reply README.md:31]: oops sorry :) wrong line... check next line

**Review by gokumar — APPROVED** [review]

**Pull request merged**